Fix typo in /using-fleet/faq (#6669)
fleetdm/fleet: docs/Using-Fleet/FAQ.md

# Using Fleet FAQ

> - [How can I switch to Fleet from Kolide Fleet?(#how-can-i-switch-to-fleet-from-kolide-fleet)

Reviewer: ```suggestion
- [How can I switch to Fleet from Kolide Fleet?](#how-can-i-switch-to-fleet-from-kolide-fleet)
```

Reviewer: @Desmi-Dizney, the link was missing the closing square bracket. I have added it for you 👍🏻

> - [What do I need to do to switch from Kolide Fleet to new Fleet?](#waht-do-i-need-to-do-to-switch-from-kolide-fleet-to-fleetdm-fleet)

Author: To migrate to Fleet from Kolide/Fleet, please follow the steps outlined in the [Upgrading Fleet section](http://localhost:2024/docs/deploying/upgrading-fleet) of the documentation.

Reviewer: Question becomes:

How do I switch to Fleet from Kolide Fleet?

Answer (from line 36 onwards) becomes what you have commented above.

Author: Hey @mike-j-thomas, I caught that the section was repeating itself. I suggest we not update the text on line 40 but delete it entirely and include the link to the other doc in the first sentence. I also searched the document for any other references to Kolide to make sure I didn't miss anything.

> - [What do I need to do to switch from Kolide Fleet to new Fleet?](#waht-do-i-need-to-do-to-switch-from-kolide-fleet-to-fleetdm-fleet)

Author: ```suggestion
- [How do I switch to Fleet from Kolide Fleet?(#waht-do-i-need-to-do-to-switch-from-kolide-fleet-to-fleetdm-fleet)
```

- [How often is the software inventory updated?](#how-often-is-the-software-inventory-updated)
- [Can I group results from multiple hosts?](#can-i-group-results-from-multiple-hosts)

> ## Upgrading Fleet

Author: ```suggestion
## How do I switch to Fleet from Kolide Fleet?
```

> To migrate to Fleet from Kolide Fleet, please follow the steps outlined in the Upgrading Fleet section of the documentation.

Author: ```suggestion
To migrate to Fleet from Kolide Fleet, please follow the steps outlined in the [Upgrading Fleet section](../Deploying/Upgrading-Fleet.md) of the documentation.
```

This makes more sense to me as a reader.

> Minor version upgrades in Kolide Fleet often included database migrations and the recommendation to back up the database before migrating. The same goes for the new Fleet.

Author: ```suggestion
```

> To migrate from Kolide Fleet to the new Fleet, please follow the steps outlined in the [Upgrading Fleet section](../Deploying/Upgrading-Fleet.md) of the documentation.

Author: ```suggestion
```
This seems weird to have the same information in the same section. See my suggestions.
